recipes/dnaio/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.3" %}
+
+package:
+  name: dnaio
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/1f/be/73d03ccdc8f721b938c5201d3031c676fa164fc6641f6d722160e5b63036/dnaio-0.3.tar.gz
+  sha256: 47e4449affad0981978fe986684fc0d9c39736f05a157f6cf80e54dae0a92638
+
+build:
+  number: 0
+  script: pip install --no-deps .
```

Comment:
can you use it like that: http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip%20install#use-pip

Reply:
Absolutely